solcore/optics/rcwa.py

```diff
@@ -57,7 +58,7 @@ def solve_rcwa(solar_cell, options):
                                             dist=position, theta=options.theta, phi=options.phi, pol=options.pol)
 
     # With all this information, we are ready to calculate the differential absorption function
-    diff_absorption, all_absorbed = calculate_absorption_rcwa(out)
+    diff_absorption, all_absorbed = calculate_absorption(out)
```

Comment:
PR #36 is adding some extra features to the `calculate_absorption` and `absorbed` functions in `rcwa.py` file, so I would't touch these, for now. Once your PR and PR #36 are merged, we can reorganise the code and move all common functions to `optics_utilities.py` as we discussed. 

In other words, keep the functions in `optics_utilities.py`, so your solver uses them, but leave the `rcwa.py` file as it was before, for now.

Reply:
Ok, I have reverted the rcwa.py file to the master branch.